src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -182,6 +183,8 @@ public class TxPool : ITxPool, IDisposable
 
         public int GetPendingBlobTransactionsCount() => _blobTransactions.Count;
 
+        public ConcurrentDictionary<string, List<Hash256>> GetBlobIndex() => _blobTransactions.GetBlobIndex;
```

Comment:
This breaks encapsulation, IMO tx pool should have a method that given blobhash, will return blob data